src/purva/NsStaticReturn.java

```diff
@@ -0,0 +1,67 @@
+package purva;
+
+public class NsStaticReturn {
+	
+	int answer;
+	
+	/*double sum(double number1, double number2, double number3){
+		double sumAnswer = number1 + number2 + number3;
+		return sumAnswer;
+		}*/
+	
+	double sumAverage(double number1, double number2, double number3) {
+		double sumAnswer = number1 + number2 + number3;
+		double avgAnswer = sumAnswer/3;
+		
+		return avgAnswer;
+	}
+	
+	boolean isAnswerAboveExpectation(double returnavgAnswer)
+	{
+		if(returnavgAnswer == 50) 
+			return true;
+		else
+			return false;
+	}
+	
+	static boolean isEligible(double avgAnswer) 
+	{
+		if (avgAnswer == 50 || avgAnswer%2==0) {
```

Comment:
we need to use divisible function not modular.
if value is more than 50 or divisible by 2. Eligibility will be printed in main method.